pkg/products/marin3r/rateLimitService.go

```diff
@@ -354,7 +368,16 @@ func (r *RateLimitServiceReconciler) getRedisSecret(ctx context.Context, client
 
 // uniqueKey generates a unique string for each possible rate limit configuration
 // combination
-func uniqueKey(r *marin3rconfig.RateLimitConfig) string {
+func uniqueKey(r marin3rconfig.RateLimitConfig) string {
 	str := fmt.Sprintf("%s/%d", r.Unit, r.RequestsPerUnit)
 	return fmt.Sprintf("%x", md5.Sum([]byte(str)))
 }
+
+func GetRateLimitFromConfig(c *corev1.ConfigMap) (*yamlRateLimit, error) {
+	ratelimitconfig := yamlRoot{}
+	err := yaml.Unmarshal([]byte(c.Data[RateLimitingConfigMapDataName]), &ratelimitconfig)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintf("error unmarshalling ratelimiting config from configmap '%s'", c.Name))
```

Comment:
Very small but could we use `fmt.Errorf` instead ? I think there was an effort to not use `github.com/pkg/errors` quite some time ago as Go can wrap / unwrap errors. Would be good to not re-introduce it unless needed :thinking: 
* https://github.com/integr8ly/integreatly-operator/pull/275